iphone/Maps/Classes/CustomViews/MapViewControls/TrafficButton/MWMTrafficButtonViewController.mm

```diff
@@ -119,7 +118,7 @@ - (void)refreshAppearance
   case MWMTrafficManagerStateOutdated: btn.imageName = @"btn_traffic_outdated"; break;
   case MWMTrafficManagerStateNoData:
     btn.imageName = @"btn_traffic_on";
-    [MWMToast showWithText:L(@"traffic_data_unavailable")];
+    [[MWMToast toastWithText:L(@"traffic_data_unavailable")] show];
```

Comment:
Swift method 'Toast.toast(withText:)' uses '@objc' inference deprecated in Swift 4; add '@objc' to provide an Objective-C entrypoint